AsyncDisplayKit/ASCollectionView.h

```diff
@@ -49,6 +50,13 @@
 - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout asyncDataFetching:(BOOL)asyncDataFetchingEnabled;
 
 /**
+ * The number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called.
```

Comment:
tiny nit:  it might be clearer to say something like "how many screens away from the end of the collection view's content you have to be for batch fetching to kick in" — I initially read this as "okay, once we've scrolled `leadingScreensForBatching` more screens, we'll call this delegate method"